core/src/main/java/org/apache/accumulo/core/clientImpl/ScanAttemptsImpl.java

```diff
@@ -111,16 +110,27 @@ public void report(ScanAttempt.Result result) {
   }
 
   Map<TabletId,Collection<ScanAttemptImpl>> snapshot() {
```

Comment:
Suggest turning the comment you removed at line 114 into a method javadoc. Just so we now the top level goal of the method. This gives a little more context to the comments inside the method.

Reply:
Added in a7043ff